CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.5
+
+* Fix tests due to new `doesNotComplete` matcher in package:test 
```

Comment:
Generally we don't add CHANGELOG entries for changes that don't affect downstream consumers.